CharacterCreator/CharacterCreator/Models/CharacterContext.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using CharacterCreator.Models;
+using Microsoft.EntityFrameworkCore;
+
+
+
+namespace CharacterCreator.Models
+{
+    public class CharacterContext : DbContext
+    {
+        public CharacterContext(DbContextOptions<CharacterContext> options) : base(options)
+        {
+        }
+
+
+        public DbSet<Characters> Features { get; set; }
```

Comment:
Characters, not Features. Also in the future, you'd normally want to keep your entity names singular (e.g. Character, Race, Class, etc) per EF Core standards

Reply:
Fixed